R/workhorse.R

```diff
@@ -1,8 +1,9 @@
-build_request <- function(endpoint = NULL,
+build_request <- function(endpoint = NULL ,
+                          path = "drive/v3/files",
                           params = list(),
                           token = NULL,
                           send_headers = NULL,
-                          api_url = NULL,
+                          api_url = "https://www.googleapis.com",
```

Comment:
This could still be `.drive$base_url <- "https://www.googleapis.com"` but, as I said, I can't exactly say why I would store it in `.drive`